javascript/builtins/WebAssembly.json

```diff
@@ -124,6 +124,168 @@
             }
           }
         },
+        "Global": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Global",
+            "support": {
+              "webview_android": {
```

Comment:
You know now that I think about it, we should probably start setting a good example on alphabetically sorting the browsers 😳

This comment applies to all three features here

Reply:
Yeah, you're right. :blush: I've updated this.